src/containers/signup/Signup.tsx

```diff
@@ -1,30 +1,44 @@
 import React from 'react';
-import './signup.less';
-import { Button, Col, Form, Input, Row, Typography } from 'antd';
-import { ParagraphProps } from 'antd/lib/typography/Paragraph';
-import { Link } from 'react-router-dom';
+import { Col, Row, Typography } from 'antd';
 import { Helmet } from 'react-helmet';
 import GreetingContainer from '../../components/greeting-container/GreetingContainer';
 import { signup } from '../../auth/ducks/thunks';
 import { connect, useDispatch } from 'react-redux';
 import { UserAuthenticationReducerState } from '../../auth/ducks/types';
 import { C4CState } from '../../store';
-import { BLACK, TEXT_GREY } from '../../colors';
+import { WHITE, BLACK, LIGHT_GREY } from '../../colors';
 import styled from 'styled-components';
+import MobilePageHeader from '../../components/mobile-pageheader/MobilePageHeader';
+import SignupForm from '../../components/signup-form/SignupForm';
+import useWindowDimensions, {
+  WindowTypes,
+} from '../../components/window-dimensions';
 
-const { Paragraph, Title } = Typography;
+const { Title } = Typography;
 
-const Footer: typeof Paragraph = styled(Paragraph)<ParagraphProps>`
-  color: ${TEXT_GREY};
-  line-height: 1.5;
+const SignupContainer = styled.div`
+  padding: 8%;
```

Comment:
⛏️ Padding 8% seems a little odd? Usually padding is fixed, usually you dont want something that changes the layout of the whole screen to be dyanmic like this because it makes the browser do a lot of redraws.

Reply:
You're right, I can fix this - originally did it this way because the padding needs to change for the different window types and this seemed a bit easier than using the hook